mos-platform/geos-cbm/geos_constants.h

```diff
@@ -0,0 +1,757 @@
+/***********************************************************************
+ * geos_constants.h
+ *
+ * This file contains #defines for use in GEOS applications
+ * with llvm-mos C compiler.
+ *
+ * Transcribed from geoProgrammer manual by DiSc.
+ *
+ ***********************************************************************/
+
+#ifndef GEOS_CONSTANTS_H
+#define GEOS_CONSTANTS_H
+
+/***********************************************************************
+ * Miscellaneous defines
+ ***********************************************************************/
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#if 0
+typedef unsigned char uint8_t;              /* an unsigned byte */
+typedef unsigned int uint16_t;              /* an unsigned 16 bit word */
+typedef int int16_t;                        /* a signed 16-bit word */
+typedef signed char int8_t;                 /* a signed byte */
+
+#define bool _Bool
+#define true 1
+#define false 0
+#endif
+
+#define LOW_BYTE(x) (*((unsigned char *)&(##x) + 0))
+#define HIGH_BYTE(x) (*((unsigned char *)&(##x) + 1))
+
+#define WORD(w) (w & 0xff), (w >> 8)
+
+#define AUX(x) #x
+#define STRINGIFY(x) AUX(x)
+
+/***********************************************************************
+ * Hardware related defines
+ ***********************************************************************/
+
+/* values for CPU_DATA - C64 memory config */
+
+#define RAM_64K 0x30        /* 64K RAM */
```

Comment:
If instead using `/** ... */` (double `**`) for comments, Doxygen will pick it up.